giveOutput.js

```diff
@@ -0,0 +1,21 @@
+const readlineSync = require('readline-sync');
+
+function decideOutput(accounts) {
+    let command = readlineSync.question('List All or List [name]? ');
+
+    if (command === 'List All') {
+        for (let i = 0; i < accounts.length; i++) {
+            delete accounts[i].Transactions;
+        }
+        console.log(accounts);
+    }
+
+    for (let i = 0; i < accounts.length; i++) {
+        if (command === 'List ' + accounts[i].Name) {

```

Comment:
As before, using a dictionary for your accounts means that instead of looping through all the accounts you can pull the name out of the command string and `console.log(accounts[name])`.